moveit_plugins/moveit_fake_controller_manager/src/moveit_fake_controller_manager.cpp

```diff
@@ -115,6 +115,12 @@ class MoveItFakeControllerManager : public moveit_controller_manager::MoveItCont
           controllers_[name].reset(new InterpolatingController(name, joints, pub_));
         else
           ROS_ERROR_STREAM("Unknown fake controller type: " << type);
+
+        moveit_controller_manager::MoveItControllerManager::ControllerState state;
+        state.default_ = controller_list[i].hasMember("default") ? (bool)controller_list[i]["default"] : false;
+        state.active_ = true;
```

Comment:
I'm not sure about the `active_` flag. Shouldn't this become true only if the controller is activated?

Reply:
All the fake controllers are always active as they can be used without prior "activation".